connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -648,7 +636,6 @@ private static InputStream getInputStream(final CloseableHttpResponse response)
             @Override
             public void close() throws IOException {
                 response.close();
-                super.close();
             }
```

Comment:
So I've narrowed this down to be an issue with the proxies, if I don't use a proxy like in your example everything is a-ok.

Thanks again for your time, I think I'll go bother someone else now :)